src/Microsoft.ML.Transforms/MissingValueDroppingTransformer.cs

```diff
@@ -339,55 +339,4 @@ private void DropNAs<TDst>(ref VBuffer<TDst> src, ref VBuffer<TDst> dst, InPredi
             }
         }
     }
-    /// <summary>
-    /// Drops missing values from columns.
-    /// </summary>
-    public sealed class MissingValueDroppingEstimator : TrivialEstimator<MissingValueDroppingTransformer>
```

Comment:
Don't we need this estimator anymore? Is there any alternative?

Reply:
We don't have extensions for this estimator. 
We use transformer as part of other transformers, but I don't see any work with estimator.
I'm tempted to make transformer internal as well, since I don't see how it can be useful for user...

---
In reply to: [265372049](https://github.com/dotnet/machinelearning/pull/2950#discussion_r265372049) [](ancestors = 265372049)

Reply:
Unless we have a scenario I would certainly welcome internalizing it, since we can always de-internalize it once we do have a scenario. But if we view this as too far then I'd understand that as well.